cli/cmd/generate.go

```diff
@@ -146,9 +146,6 @@ func runGenerate(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("failed to unmarshal manifest: %w", err)
 	}
 	manifest.Policies = policyMap
-	if err := manifest.Validate(); err != nil {
-		return fmt.Errorf("validating manifest: %w", err)
-	}
```

Comment:
I think we should ask the user to fill out the missing required fields based on the platform we are targeting.

Reply:
Did you have anything concrete in mind? Should we just print some instructions to stdout?

Reply:
Check out the way we print output for the user a few lines above. Output should be indented by 2 spaces.